reporter/src/test/kotlin/reporters/NoticeReporterTest.kt

```diff
@@ -57,7 +57,8 @@ class NoticeReporterTest : WordSpec({
             resultFile.readText() shouldBe expectedResultFile.readText()
         }
 
-        "not contain licenses of excluded packages" {
+        // The test will be enabled again after re-implementing the exclusion logic in NoticeReporter.
```

Comment:
Maybe mark this as a "TODO" comment for more visibility?

Reply:
Good point, but I will leave it as is because the PR which will re-activate the test is almost ready.